src/main/scala/mesosphere/marathon/core/readiness/ReadinessCheckExecutor.scala

```diff
@@ -12,6 +12,9 @@ import scala.concurrent.duration.FiniteDuration
 /**
   * Poll readiness of the given endpoint until we receive readiness confirmation.
   * Intermediate results are returned as part of the Observable.
+  *
+  * Readiness checks are currently only available for AppDefinitions, therefore this code
```

Comment:
we should probably add Readiness checks to the pods and then have it return an empty sequence.